crossbeam/xm_train.py

```diff
@@ -48,8 +48,8 @@ def main(argv) -> None:
   uname = getpass.getuser()
 
   with xm_abc.create_experiment(experiment_title=_EXP_NAME.value) as experiment:
-    job_requirements = xm.JobRequirements(ram=26 * FLAGS.num_gpus * xm.GiB,
-                                          cpu=7 * FLAGS.num_gpus,
+    job_requirements = xm.JobRequirements(ram=8 * FLAGS.num_gpus * xm.GiB,
```

Comment:
Is it needed to reduce our resource consumption here? We can request the amount we were using before: https://g3doc.corp.google.com/company/teams/brain-frameworks/xcloud/guide.md?cl=head